src/planscape/plan/tests.py

```diff
@@ -439,8 +442,49 @@ def test_good_polygon(self):
     def test_good_multipolygon(self):
         response = self.client.post(
             reverse('plan:create_project_area'),
-            {'project_id': self.project_no_user.pk,'geometry': {'features': [
+            {'project_id': self.project_no_user.pk, 'geometry': {'features': [
                 {'geometry': {'type': 'MultiPolygon', 'coordinates': [[[[1, 2], [2, 3], [3, 4], [1, 2]]]]}}]}},
             content_type='application/json')
         self.assertEqual(response.status_code, 200)
         self.assertEqual(ProjectArea.objects.count(), 1)
+
+
+class GetProjectTest(TransactionTestCase):
+    def setUp(self):
+        self.user = User.objects.create(username='testuser')
```

Comment:
Since there is a "user", can you add a few tests to check when the user is logged in?

Reply:
Added mirroring tests for a signed in user, let me know if you think we could use more